### PR TITLE
kv (ticdc): reduce some verbose log

### LINKED ISSUE #11074: There are too many resolve lock logs in cdc log.
### What did you do?

create a changefeed replicate sysbench workload.

### What did you expect to see?

Not too many verbose logs.

### What did you see instead?

There are too many resolve lock logs in cdc log.

```
[2024/05/08 19:46:40.263 +08:00] [INFO] [lock_resolver.go:63] ["resolve lock finishes"] [regionID=33516] [lockCount=0] [maxVersion=449621204473741312] [namespace=default] [changefeed=ticdc-task] []
[2024/05/08 19:46:40.263 +08:00] [INFO] [lock_resolver.go:56] ["resolve lock starts"] [regionID=33520] [maxVersion=449621204473741312] [namespace=default] [changefeed=ticdc-task]
[2024/05/08 19:46:40.263 +08:00] [INFO] [lock_resolver.go:63] ["resolve lock finishes"] [regionID=33520] [lockCount=0] [maxVersion=449621204473741312] [namespace=default] [changefeed=ticdc-task] []


> grep "resolve lock" ticdc-2024-05-08T19-48-52.508.log| wc -l
  378604
> wc -l ticdc-2024-05-08T19-48-52.508.log
  926626 ticdc-2024-05-08T19-48-52.508.log
```

### Versions of the cluster

Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

```console
(paste TiDB cluster version here)
```

Upstream TiKV version (execute `tikv-server --version`):

```console
(paste TiKV version here)
```

TiCDC version (execute `cdc version`):

```console
(paste TiCDC version here)
```

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve?

Issue Number: close #11074 

#### Tests 

 - No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/kv/shared_client.go [hunk 1]
```diff
@@ -815,7 +815,7 @@ func (s *SharedClient) logSlowRegions(ctx context.Context) error {
 			return ctx.Err()
 		case <-ticker.C:
 		}
-		log.Info("event feed starts to check locked regions",
```
Comment (hicqu) [inline cdc/kv/shared_client.go:818]: There shouldn't be too many such logs.

**Comment by hicqu** [comment]
Rest LGTM.

File: cdc/kv/shared_client.go (hunk 1 above)
Comment (asddongmen) [reply cdc/kv/shared_client.go:818]: It depends on the number of changefeed, if there hundreds of changefeed, there will be a lot of this log.

**Review by sdojjy — APPROVED** [review]

**Review by hicqu — APPROVED** [review]

**Pull request merged**